### PR TITLE
feat: import styletron post + blog styling updates

### PR DESCRIPTION
**Pull request by jh3y**
#### Description
- imports `Getting started with Styletron` post
- introduces styling changes for blog
- introduce image captions in order to credit Ryan with infographic

#### Scope
- [ ] Patch: Bug Fix
- [ ] Minor: New Feature
- [ ] Major: Breaking Change

### CODE REVIEW

**Comment by gergelyke** [comment]
can we order the posts based on the publish date?

**Comment by jh3y** [comment]
@gergelyke Of course 👍 Not sure how I overlooked that 😅 

I've updated the post generator script to order them by date.

File: documentation-site/pages/blog/getting-started-with-base-web/index.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import metadata from './metadata.json'
 
 export default Layout;
 
-<BlogImage style={{marginTop: '20px'}} src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" />
+<BlogImage style={{marginTop: '20px'}} full src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" caption='Base UI by Uber Design'/>
```
Comment (nadiia) [inline documentation-site/pages/blog/getting-started-with-base-web/index.mdx]:
```suggestion
<BlogImage style={{marginTop: '20px'}} full src="https://cdn.dribbble.com/users/914217/screenshots/4615198/base_ui.png" caption='Base Web by Uber Design'/>
```

File: documentation-site/pages/blog/getting-started-with-styletron/index.mdx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+There are many CSS in JS libraries available. Each have their own take on how CSS in JS should be. Styletron is one of those libraries.
+Valerii Sorokobatko has curated some of the popular options in this
+Github repo: [awesome-css-in-js](https://github.com/tuchk4/awesome-css-in-js).
+
+## Styletron
+
+So what makes Styletron any different? It boasts high performance and scalability. It's component-oriented. But these aren't what sets it apart.
+
+The USP for Styletron is its novel approach to CSS in JS. Other libraries do things like inline styles or generate scoped classes.
```
[the remark this answers is not in the document]
Comment (jh3y) [reply documentation-site/pages/blog/getting-started-with-styletron/index.mdx]: Unique selling point 👍

File: documentation-site/pages/blog/getting-started-with-base-web/index.mdx (hunk 1 above)
Comment (jh3y) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: I wasn't sure about this one as the actual title on dribbble is Base UI. Base Web being the implementation of Base UI right?

Comment (nadiia) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: I think they made it just `Base` that has `BaseWeb`/`BaseMobile` @gergelyke is that correct?

Comment (gergelyke) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: that's correct - everything we work on in this repository is Base Web

Comment (jh3y) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: The point I'm getting at is that the actual `dribble` shot title is `Base UI`. It's not a picture of `Base Web`. The suggested change is to update the caption.

Comment (gergelyke) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: Base UI got rebranded to Base Web, so they are the same things, but moving forward, we'll use the Base Web name. Does it make sense?

Comment (jh3y) [reply documentation-site/pages/blog/getting-started-with-base-web/index.mdx]: Yeah that makes sense. I wasn't aware of that 👍

**Pull request merged**